### PR TITLE
feat: Add support for Person + Event credentials [DEV-1918]

### PR DESCRIPTION
**Pull request by DaevMithran**

### CODE REVIEW

File: src/api/controllers/credentials.ts [hunk 1]
```diff
@@ -83,23 +83,45 @@ export class Credentials {
...
+					'@context': VC_CONTEXT.concat(VC_EVENTRESERVATION_CONTEXT),
+					type: ['EventReservation', VC_TYPE],
+					issuanceDate: new Date().toISOString(),
+					credentialSubject: credential_subject,
+					reservationId: new URL(request.url).searchParams.get('data'),
+					reservationStatus: 'https://schema.org/ReservationConfirmed',
+					reservationFor: {
+						'@type': 'Event',
+						name: 'IIW Event',
+						startDate: "2022-11-06T19:30:00-08:00",
```
Comment (ankurdotb) [inline src/api/controllers/credentials.ts:100]: Wrong fixed value, correct to actual

Comment (ankurdotb) [inline src/api/controllers/credentials.ts:100]: Check ClickUp for ticket body

File: src/api/controllers/credentials.ts [hunk 2]
```diff
@@ -83,23 +83,45 @@ export class Credentials {
...
+					'WebPage': [
+						{
+							'@type': 'ProfilePage',
+							description: 'Twitter',
+							name: `${user?.nickname}` ?? '<unknown>',
+							identifier: `@${user?.handle}` ?? '<unknown>',
+							URL: `https://twitter.com/${user?.handle}`,
+							lastReviewed: user?.updated_at
+						}
+					],
```
Comment (ankurdotb) [inline src/api/controllers/credentials.ts:121]: Modify to handle Twitter, Discord etc dynamically

File: src/api/mixins/guard.ts [hunk 3]
```diff
@@ -14,7 +14,7 @@ export class GuardedCredentials {
 					body: JSON.stringify(
 						{
 							claim,
-							provider
+							provider: 'twitter'
 						}
```
Comment (ankurdotb) [inline src/api/mixins/guard.ts:18]: Handle different upstreams correctly

**Review by ankurdotb — CHANGES_REQUESTED** [review]

**Comment by ankurdotb** [comment]
Task linked: [DEV-1896 Deploy credential service and auth0 changes to staging for IIW demo changes](https://app.clickup.com/t/6600954/DEV-1896)

**Review by ankurdotb — CHANGES_REQUESTED** [review]
Resolve comments + discussion yesterday

File: src/api/controllers/credentials.ts (hunk 1 above)
Comment (DaevMithran) [reply src/api/controllers/credentials.ts]: @ankurdotb updated this, can you have a look

File: src/api/controllers/credentials.ts [hunk 4]
```diff
@@ -83,24 +83,48 @@ export class Credentials {
...
+					'WebPage': [
+						{
+							'@type': 'ProfilePage',
+							description: provider,
+							name: `${user?.nickname}` ?? '<unknown>',
+							identifier: `@${user?.nickname}` ?? '<unknown>',
+							URL: `https://twitter.com/${user?.nickname}`,
+							lastReviewed: user?.updated_at
+						}
+					],
```
Comment (ankurdotb) [inline src/api/controllers/credentials.ts:123]: This needs to be made more generic than just Twitter

**Pull request merged**